modules/kubernetes-addons/kyverno/kyverno-ui-values.yaml

```diff
@@ -0,0 +1,11 @@
+ui:
```

Comment:
since there are only charts for `kyverno` and `kyverno-policies`, we should remove this additional `kyverno-ui` component and match the upstream project https://docs.aws.amazon.com/eks/latest/userguide/launch-templates.html

users can pass in the appropriate values to enable the UI component through the `kyverno` module

Reply:
ah, ok now I see - they split out a separate project https://github.com/kyverno/policy-reporter/tree/main/charts/policy-reporter

we should keep the names consistent and instead call this `kyverno-policy-reporter` throughout